Makefile

```diff
@@ -273,22 +275,9 @@ buildx: bundles/buildx ## build buildx cli tool
 endif
 endif
 
-# This intentionally is not using the `--output` flag from the docker CLI, which
-# is a buildkit option. The idea here being that if buildx is being used, it's
-# because buildkit is not supported natively
 bundles/buildx: bundles ## build buildx CLI tool
-	docker build -f $${BUILDX_DOCKERFILE:-Dockerfile.buildx} -t "moby-buildx:$${BUILDX_COMMIT:-latest}" \
-		--build-arg BUILDX_COMMIT \
-		--build-arg BUILDX_REPO \
-		--build-arg GOOS=$$(if [ -n "$(GOOS)" ]; then echo $(GOOS); else go env GOHOSTOS || uname | awk '{print tolower($$0)}' || true; fi) \
-		--build-arg GOARCH=$$(if [ -n "$(GOARCH)" ]; then echo $(GOARCH); else go env GOHOSTARCH || true; fi) \
-		.
-
-	id=$$(docker create moby-buildx:$${BUILDX_COMMIT:-latest}); \
-	if [ -n "$${id}" ]; then \
-		docker cp $${id}:/usr/bin/buildx $@ \
-		&& touch $@; \
-		docker rm -f $${id}; \
-	fi
-
-	$@ version
+	goarch="$$(uname -m | sed -e 's/x86_64/amd64/g')"; \
+	goos="$$(uname -s | awk '{print tolower($$0)}')"; \
+	curl -o $@ -fsSL "https://github.com/docker/buildx/releases/download/${BUILDX_VERSION}/buildx-${BUILDX_VERSION}.$${goos}-$${goarch}" \
```

Comment:
Discussing with @tonistiigi @cpuguy83 and opting to not do the automatic detection, but to default to ppc64le (which is what we use it for), and a make variable to override.

Reply:
We can use the script from https://github.com/moby/buildkit/blob/master/hack/install-buildx (needs some minor changes to allow overriding the parameters)